.travis.yml

```diff
@@ -241,18 +110,26 @@ script:
   - "printf 'packages: transformers-compat-*/*.cabal transformers-compat-tests-*/*.cabal\\n' > cabal.project"
   - cat cabal.project
 
+  # Build with installed constraints for packages in global-db
+  - if $INSTALLED; then echo cabal new-build -w ${HC} --disable-tests --disable-benchmarks $(${HCPKG} list --global --simple-output --names-only | sed 's/\([a-zA-Z0-9-]\{1,\}\) */--constraint="\1 installed" /g') all | sh; else echo "Not building with installed constraints"; fi
 
   # build & run tests, build benchmarks
-  - cabal new-build -w ${HC} ${CABALFLAGS} ${TEST} ${BENCH} -j2 all
-  - if [ "x$TEST" = "x--enable-tests" ]; then cabal new-test -w ${HC} ${CABALFLAGS} ${TEST} ${BENCH} all; fi
+  - cabal new-build -w ${HC} ${TEST} ${BENCH} all
```

Comment:
This removes the `-j2` flag. Please add it back.

Reply:
Interesting.

This doesn't pertain to the PR itself, but would it make sense to have `--jobs=2` be the default for `haskell-ci`? It appears that it was the default previously (in the sense that it would always generate `-j2` to begin with), and it might save us some brain cells if we don't have to remember this every time we regenerate the script.

Reply:
IIRC it did `-j2` for dependencies only. As for package itself `-j2` is little use (in fact it does nothing as there are hardly components to paralellize over - it's not `ghc-options: -j2`!)

(it's beneficial for e.g .`servant`, where we have many independent cookbook recipe-packages - but not for `transformers-compat`).

So I don't know what to do here actually.